src/comms.cpp

```diff
@@ -61,6 +61,8 @@ void clover_barrier() {
 }
 
 void clover_barrier(global_variables &globals) {
+	#pragma omp flush
```

Comment:
This should only do host memory flushing. Why do you need this?

Reply:
Removed, see https://github.com/UoB-HPC/cloverleaf_openmp_target/pull/2/commits/030edc6cb280ba3f773a581bbd53393e3d9828b3